.github/workflows/publish-cli-bins.yaml

```
@@ -9,74 +9,136 @@
   pull_request:
 
 jobs:
+  setup:
+    name: Set Up Shared Environment
+    runs-on: ubuntu-latest
+    steps:
+    - name: Cache apt packages
+      uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)